source/MapDetailPanel.cpp

```diff
@@ -488,57 +533,59 @@ void MapDetailPanel::DrawInfo()
 				const Planet *planet = object.GetPlanet();
 				if(planet->IsWormhole() || !planet->IsAccessible(player.Flagship()) || shown.count(planet))
 					continue;
-				shown.insert(planet);
-
-				SpriteShader::Draw(planetSprite, uiPoint);
-				planetY[planet] = uiPoint.Y() - 60;
-
-				font.Draw({object.Name(), alignLeft},
-					uiPoint + Point(-70., -52.),
-					planet == selectedPlanet ? medium : dim);
-
-				bool hasSpaceport = planet->HasSpaceport();
-				string reputationLabel = !hasSpaceport ? "No Spaceport" :
-					GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
-					planet->GetGovernment()->IsEnemy() ? "Hostile" :
-					planet->CanLand() ? "Friendly" : "Restricted";
-				font.Draw(reputationLabel,
-					uiPoint + Point(-60., -32.),
-					hasSpaceport ? medium : faint);
-				if(commodity == SHOW_REPUTATION)
-					PointerShader::Draw(uiPoint + Point(-60., -25.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
-
-				font.Draw("Shipyard",
-					uiPoint + Point(-60., -12.),
-					planet->HasShipyard() ? medium : faint);
-				if(commodity == SHOW_SHIPYARD)
-					PointerShader::Draw(uiPoint + Point(-60., -5.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
-
-				font.Draw("Outfitter",
-					uiPoint + Point(-60., 8.),
-					planet->HasOutfitter() ? medium : faint);
-				if(commodity == SHOW_OUTFITTER)
-					PointerShader::Draw(uiPoint + Point(-60., 15.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
-
-				bool hasVisited = player.HasVisited(*planet);
-				font.Draw(hasVisited ? "(has been visited)" : "(not yet visited)",
-					uiPoint + Point(-70., 28.),
-					dim);
-				if(commodity == SHOW_VISITED)
-					PointerShader::Draw(uiPoint + Point(-70., 35.), Point(1., 0.),
-						10.f, 10.f, 0.f, medium);
-
-				uiPoint.Y() += 130.;
+
+				if((scroll - 65.) / 130. <= planetNbr && uiPoint.Y() + int(scroll) % 130 < Screen::Bottom() - 295)
+				{
+					shown.insert(planet);
+
+					if(planet == selectedPlanet)
+						FillShader::Fill(Point(Screen::Left() + 110., uiPoint.Y()), Point(220., 110.), faint);
+
+					bool hasSpaceport = planet->HasSpaceport();
+
+					const Sprite *sprite = object.GetSprite();
+					double scale = min(.5f, min((100) / sprite->Width(), (100) / sprite->Height()));
+					SpriteShader::Draw(sprite, Point(Screen::Left() + 50, uiPoint.Y()), scale);
+					planetY[planet] = uiPoint.Y() - 65;
+
+					font.Draw({ object.Name(), alignLeft }, uiPoint + Point(0., -52.),
+						planet == selectedPlanet ? medium : dim);
+
+					string reputationLabel = !hasSpaceport			 ? "No Spaceport" :
+						GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
+						planet->GetGovernment()->IsEnemy()			 ? "Hostile" :
+						planet->CanLand()							 ? "Friendly" :
```

Comment:
What's going on with your indentation here?

Reply:
I guess the code "cleanup" broke some things